src/pages/coviidUpdatedPage/index.js

```diff
@@ -24,18 +34,32 @@ const cardsText = [
 ]
 
 const CoviidUpdated = () => {
+  const location = useLocation()
+  console.log(location)
+
+  const consented = location.pathname.includes(
```

Comment:
get the value of the route param value from `useParam` and check whether its consented or not. either true or false. if its not either then default to false for now. does that make sense?